opfpush/src/main/java/org/onepf/opfpush/model/NotificationPayload.java

```diff
@@ -66,36 +67,73 @@ private NotificationPayload(@NonNull final String title,
         this.clickAction = clickAction;
     }
 
+    /**
+     * Returns the field that indicates notification title.
+     *
+     * @return The field that indicates notification title.
+     */
     @NonNull
     public String getTitle() {
         return title;
     }
 
+    /**
+     * Returns the field that indicates notification icon.
+     *
+     * @return The field that indicates notification icon.
+     */
     @NonNull
     public String getIcon() {
         return icon;
     }
 
+    /**
+     * Returns the field that indicates notification body text.
+     *
+     * @return The field that indicates notification body text.
+     */
     @Nullable
     public String getBody() {
         return body;
     }
 
+    /**
+     * Returns the field that indicates sound to be played.
+     *
+     * @return The field that indicates sound to be played. Supports only {@code default} currently.
```

Comment:
>> can't parse
It's copypaste too. It's mean that you can put only "default" string as parameter. Other values will be ignored.